.env

```diff
@@ -0,0 +1,14 @@
+DB_NAME = 'ddibi6hgtgr7ol'
```

Comment:
please ignore this file , please

Reply:
I forgot to do that, but it may not happen again